client/globals.d.ts

```diff
@@ -123,10 +123,7 @@ declare global {
 		isNextDepositNoticeDismissed: boolean;
 		isInstantDepositNoticeDismissed: boolean;
 		isDateFormatNoticeDismissed: boolean;
-		reporting: {
-			exportModalDismissed?: boolean;
-		};
-		locale: {
+		userLocale: {
 			code: string;
 			english_name: string;
 			native_name: string;
```

Comment:
Another good opportunity to gift extra information to future devs 🎁

This is particularly helpful for `code`, because it is specific to the transact-platform-server, not WP-standard locale as one might expect.

```suggestion
		userLocale: {
			/**
			 * The locale of the current user profile, represented as a locale code supported by transact-platform-server.
			 *
			 * @example 'sv' // Swedish
			 *
			 * @see WC_Payments_Utils::convert_to_server_locale
			 */
			code: string;

			/**
			 * The English name of the locale.
			 *
			 * @example 'Swedish'
			 */
			english_name: string;

			/**
			 * The native name of the locale.
			 *
			 * @example 'Svenska'
			 */
			native_name: string;
```

Reply:
I have added the comments. This will be certainly helpful for future developers - [7b4f520](https://github.com/Automattic/woocommerce-payments/pull/10003/commits/7b4f5208ecbe35b593e1ce98800293ec7bb11284)